torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -960,3 +990,76 @@ def test_call_fork_in_jit_with_profiling(self):
         events = prof.function_events
         function_event = get_function_event(events, "foo")
         self.assertEqual(function_event.name, "foo")
+
+
+class JitFaultyAgentRpcTest(FaultyRpcAgentTestFixture):
```

Comment:
Need to add this TestCase to class JitRpcTest as well, otherwise the test is not run.

Also, no need to inherit from FaultyRpcAgentTestFixture. JitRpcTest does that for us.